address/deployment.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: address
+  labels:
+    app: address
+    hail.is/sha: "{{ code.sha }}"
+spec:
+  selector:
+    matchLabels:
+      app: address
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: address
+        hail.is/sha: "{{ code.sha }}"
+        grafanak8sapp: "true"
```

Comment:
Do we need this?

Reply:
I copied this from batch. It should probably be removed. Let's remove all of them in one separate PR.